Source/WebKit/WebProcess/Extensions/API/Cocoa/WebExtensionAPIAlarmsCocoa.mm

```diff
@@ -85,19 +85,13 @@
 {
     // Documentation: https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/alarms/create
 
-    static NSArray<NSString *> *optionalKeys = @[
-        whenKey,
-        delayInMinutesKey,
-        periodInMinutesKey,
-    ];
-
     static NSDictionary<NSString *, id> *types = @{
         whenKey: NSNumber.class,
         delayInMinutesKey: NSNumber.class,
         periodInMinutesKey: NSNumber.class,
     };
 
-    if (!validateDictionary(alarmInfo, @"info", nil, optionalKeys, types, outExceptionString))
+    if (!validateDictionary(alarmInfo, @"info", nil, types, outExceptionString))
```

Comment:
Are there no required keys for this method? What happens if you pass an empty object?

Reply:
Nothing required, it defaults to a 1 minute alarm.